docs/src/Checkbox.doc.js

```diff
@@ -193,42 +193,98 @@ card(
   <Example
     id="refExample"
```

Comment:
@AlbertCarreras fyi changed this back to `refExample` - we use the ID to link to specific sections of the page
![image](https://user-images.githubusercontent.com/127199/89352388-d82d1f00-d668-11ea-86f9-3141c2cd5cb5.png)